src/utils/sftp/handle-image-forms.ts

```diff
@@ -0,0 +1,178 @@
+import toast from "react-hot-toast";
+import { env } from "~/env.mjs";
+import { handleCreateSftpFile } from "~/utils/sftp/handle-create-sftp-file";
+import { handleDeleteSftpFile } from "~/utils/sftp/handle-delete-sftp-file";
+
+type ImageOperationResult<T> = {
+  data?: T;
+  error?: string;
+  success: boolean;
+};
```

Comment:
Tror detta är en mer korrekt type av vad du försöker göra. Vet dock inte om det förbättrar någon annat.
```suggestion
type ImageOperationResult<T> = {
  success: boolean;
  data: T;
} | {
  success: false;
  error: string;
};
```